pyqtgraph/graphicsItems/ROI.py

```diff
@@ -1200,14 +1200,16 @@ class Handle(UIGraphicsItem):
     sigClicked = QtCore.Signal(object, object)   # self, event
     sigRemoveRequested = QtCore.Signal(object)   # self
     
-    def __init__(self, radius, typ=None, pen=(200, 200, 220), parent=None, deletable=False):
+    def __init__(self, radius, typ=None, pen=fn.mkPen(200, 200, 220), parent=None, deletable=False,
+                 activePen = fn.mkPen(255, 255,0)):
```

Comment:
It would be better to leave the default arguments as tuples and put the calls to `mkPen()` inside the `__init__()` method.

Github pull requests always track the changes between two _branches_, so as soon as you update your develop branch, those changes are included in the PR (this allows you to continue modifying the PR until we decide it is ready to be merged). To avoid mixing unrelated changes, you should create a new topic branch for each PR.